generated-platform-project/quarkus-camel/integration-tests/camel-quarkus-integration-test-fop/pom.xml

```diff
@@ -57,7 +57,7 @@
     <dependency>
       <groupId>org.apache.pdfbox</groupId>
       <artifactId>pdfbox</artifactId>
-      <version>3.0.2</version>
```

Comment:
This is not correct but should be fine: https://quarkusio.zulipchat.com/#narrow/stream/187038-dev/topic/quarkus-platform.3A.20weird.20upgrade.20issue